Code of Conduct.md

```diff
@@ -7,7 +7,7 @@ Contact: the WHATWG Steering Group, via <sg@whatwg.org>, or a member of the comm
 * We are committed to providing a friendly, safe, and welcoming environment for all, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or other similar characteristic.
 * Please be kind and courteous. There's no need to be mean or rude.
 * Respect that people have differences of opinion and that every design or implementation choice carries a trade-off and numerous costs. There is seldom a right answer.
-* You shall not insult, demean, or harass anyone. We interpret the term "harassment" as including the definition in the [Citizen Code of Conduct](http://citizencodeofconduct.org/); if you have any lack of clarity about what might be included in that concept, please read their definition. In particular, we don't tolerate behavior that excludes people in socially marginalized groups.
+* You shall not insult, demean, or harass anyone. Harassment includes, but is not limited to, offensive verbal comments related to gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, ethnicity, religion, and technology choices; sexual images in public spaces, deliberate intimidation, stalking, doxing, following, inappropriate photography or recording, inappropriate physical contact, unwelcome sexual attention, sustained disruption of events, as well as advocating for or encouraging harassment. In particular, we don't tolerate behavior that excludes people in socially marginalized groups.
```

Comment:
Personally, I'd appreciate "nationality" also being on that list

Reply:
SGTM. @annevk do you want to pick a place in the list to insert it?

Reply:
Done. I did notice that we end up saying the same thing twice, sorta, but it's the kind of thing that's worth repeating.